bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/accessor/LookupValueAccessor.java

```diff
@@ -66,15 +66,21 @@ protected void updateControlFromValue(Control control, Value value) {
 			((Lookup) control).getDescription().setText("");
 			((Lookup) control).setText("");
 			if (((Lookup) control).getEditable()) {
-				((Lookup) control).setMessage("Null");
+				((Lookup) control).setMessage("...");
+				if (LOG) {
+					System.out.println("Lookup " + ((Lookup) control).getLabel().getText() + " ist null");
+				}
```

Comment:
Für das losging sollten wir das Standard log Framework verwenden. Output nach sysout gehört verboten ;), weil man das beim Anwender nicht nachvollziehen kann.

Reply:
Die Information, die hier geloggt wird, ist für den Anwender allerdings nicht interessant, sondern nur für uns für debugging. Entsprechend würde ich diese Information nicht in das Log File schreiben.

Reply:
Ist ok